### ISSUE TITLE
question about keep radio part

### ISSUE THREAD

**Issue by superPangpang** [Pay20Y/GCAN#4]
Thank you so much for your source code ! But I think there's something wrong with keep radio part:
     new_width = int((1.0 * H / input_height) * input_width)
I don't know if I was wrong, Looking forward to your answers.Thanks so much!

**Comment by Pay20Y** [comment]
Yes, you are right, the input_width should be replaced with W. Thanks for pointing.
